### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by craftid**

### CODE REVIEW

File: server/src/routes/index.ts [hunk 1]
```diff
@@ -34,48 +37,6 @@ if (env.NODE_ENV === "production") {
...
-} else {
-	// redirect to 5173 port
-	router.get("*", (req, res, next) => {
-		if (req.path.startsWith("/api")) {
-			return next()
-		}
-
-		res.redirect(env.PUBLIC_URL + req.path)
-	})
 }
```
Comment (arafays) [inline server/src/routes/index.ts:40]: looks cleaner

File: server/src/server.ts [hunk 2]
```diff
@@ -21,6 +21,12 @@ export const server = () => {
 	app.use(express.json())
 	app.use(helmet())
 	app.use(cors())
+
```
Comment (arafays) [inline server/src/server.ts:24]: need to improve the placement of this

File: server/src/socket.ts [hunk 3]
```diff
@@ -1,44 +1,57 @@
...
+debug.enable("express:socket")
+
 const log = debug("express:socket")
 
 export const setupSocket = (server: any) => {
-	const io = new Server(server)
+	log("setting up socket")
+	const io = new Server(server, {
+		cors: {
+			origin: "*",
```
Comment (arafays) [inline server/src/socket.ts:12]: need to make the websocket secure to not allow all origins

**Review by arafays — CHANGES_REQUESTED** [review]
can you make these change?

File: server/src/routes/index.ts (hunk 1 above)
Comment (craftid) [reply server/src/routes/index.ts:40]: thanks

**Comment by craftid** [comment]
these are base commits need to update depending on the project if a better solution that can help this being a starter we can implement them then

**Pull request merged**